debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/converter/ByteBufferConverter.java

```diff
@@ -0,0 +1,69 @@
+package io.debezium.connector.mysql.converter;
```

Comment:
This should have the standard file header (see any other Debezium file for an example). Also this should be located in the _debezium-core_ module instead of the MySQL connector.